README.md

```diff
@@ -247,19 +264,19 @@ currying, we recommend the following articles:
 - [Apple's documentation of curried functions](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Declarations.html#//apple_ref/doc/uid/TP40014097-CH34-XID_615)
 - [Introduction to Function Currying in Swift](http://robots.thoughtbot.com/introduction-to-function-currying-in-swift)
 
-The last thing to do will be to conform to `JSONDecodable` and implement the
+The last thing to do will be to conform to `Decodable` and implement the
 required `decode` function. We will implement this function by using `map`
 (`<^>`) and `apply` (`<*>`) to conditionally pass the required parameters to
-the creation function. The common pattern will look like:
+the `create` function. The common pattern will look like:
```

Comment:
maybe we should say

> implement a curried creation function

instead of making it sound like people _have_ to write a function called `create`